### PR TITLE
Store QC libs version in DB

### PR DESCRIPTION
**Pull request by doaa-altarawy**
## Description
Save in DB the versions of QCElemental and QCFractal that created the DB

## Todos
Get the libraries version in the right place.
Use the returned json when needed to compare against current ones.

## Status
- [x] Ready to go

### CODE REVIEW

File: qcfractal/storage_sockets/sqlalchemy_socket.py [hunk 1]
```diff
@@ -1939,3 +1948,27 @@ def remove_user(self, username):
...
+                elemental_version = qcelemental.__version__
+                fractal_version = qcfractal.__version__
+                engine_version = qcengine.__version__
+                current = VersionsORM(elemental_version=elemental_version,
+                                      fractal_version=fractal_version,
+                                      engine_version=engine_version)
+                session.add(current)
+                session.commit()
+            else:
+                current = db_ver.first()
```
Comment (dgasmith) [inline qcfractal/storage_sockets/sqlalchemy_socket.py]: Cool, looks good and makes sense. I can work off of this.

**Comment by dgasmith** [comment]
@doaa-altarawy Does this need to be rebased? I think otherwise ready to go.

**Comment by doaa-altarawy** [comment]
I have just rebased, it's ready to go

**Comment by dgasmith** [comment]
Waiting on the GitHub hooks to come back.

**Pull request merged**